pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -3177,13 +3177,15 @@ let
           license = lib.licenses.mit;
         };
       };
-
-    };
+    });
```

```suggestion
    };
```
Is this really necessary?

---

removed it